configs/common/urls.js

```diff
@@ -0,0 +1,38 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at https://mozilla.org/MPL/2.0/. */
+
+/* please keep keys in this object sorted */
+module.exports = (env = "sandbox") => {
+  if (env === "sandbox") {
+    return {
+      ENDPOINT_HPNV2_ANONYMOUS: "https://collector.wdp.brave.software", // hpnv2/sources/endpoints.es
+      ENDPOINT_HPNV2_DIRECT: "https://collector.wdp.brave.software", // hpnv2/sources/endpoints.es
+      ENDPOINT_PATTERNS: "https://patterns.hpn.brave.software/patterns.gz",
+      ENDPOINT_SAFE_QUORUM_PROVIDER: "https://safe-browsing-quorum.hpn.brave.software/config",
+      ENDPOINT_STAR: "https://star.wdp.brave.software/",
+      FETCHER_GATEWAY: "https://fg.search.brave.com",
```

Comment:
Should this be `brave.software`?

Reply:
At the moment we only have one of these deployed to prod and the feature is fully disabled in the WDP extension (no way to opt-in).